### ISSUE TITLE
NS5000 is not reported for ReceivedExtensions.Received calls

### ISSUE THREAD

**Issue by tpodolak** [nsubstitute/NSubstitute.Analyzers#131]
Repro
```c#
using NSubstitute;
using NSubstitute.ReceivedExtensions;

namespace MyNamespace
{
    public class FooTests
    {
        public class Foo
        {
            public int Id { get; }
        }
        
        public void Test()
        {
            var secondSubstitute = Substitute.For<Foo>();
            
            secondSubstitute.Received(Quantity.AtLeastOne());
        }
    } 
}
```
